browser/js/common/factories/StyleModeFactory.js

```diff
@@ -0,0 +1,103 @@
+app.factory("StyleModeFactory", function(StylingFactory, $compile, $rootScope){
```

Comment:
Can you add some comments around each function? Hard to keep track based on names :)